### PR TITLE
Convert worlds to ENU

### PR DESCRIPTION
**Pull request by lukicdarkoo**
- [x] Convert worlds to ENU
- [x] Fix IMU usage (sync with Webots `develop`)
- [x] Fix GPS usage (sync with Webots `develop`)
- [x] Upgrade Webots headers
- [x] Fix panoramix view

This is tightly related to:
https://github.com/cyberbotics/webots/pull/2424

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
At this point, it solves all issues from:
https://github.com/cyberbotics/webots_ros/issues/37

We can merge it after https://github.com/cyberbotics/webots/pull/2424 is merged (and maybe some cleanup)

File: src/pioneer3at.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ webots_ros::set_int timeStepSrv;
 
 static const char *motorNames[NMOTORS] = {"front_left_wheel", "front_right_wheel", "back_left_wheel", "back_right_wheel"};
 
-static double GPSValues[3] = {0, 0, 0};
+static double GPSValues[3] = {0, 0};
```
Comment (omichel) [inline src/pioneer3at.cpp:52]: That looks wrong...
```suggestion
static double GPSValues[2] = {0, 0};
```

**Review by omichel — CHANGES_REQUESTED** [review]

File: src/pioneer3at.cpp (hunk 1 above)
Comment (lukicdarkoo) [reply src/pioneer3at.cpp:52]: It is :)

**Review by omichel — APPROVED** [review]
All good. Thanks.

**Pull request merged**